include/RAJA/index/IndexSetBuilders.hpp

```diff
@@ -45,10 +56,165 @@ namespace RAJA
  *
  ******************************************************************************
  */
+template <typename WORKING_RES>
 void buildTypedIndexSetAligned(
     RAJA::TypedIndexSet<RAJA::RangeSegment, RAJA::ListSegment>& hiset,
     const Index_type* const indices_in,
-    Index_type length);
+    Index_type length)
+{
+  if (length == 0) return;
+
+  camp::resources::Resource work_res{camp::resources::WORKING_RES()};
```

Comment:
The camp::resources::WORKING_RES doesn't seem like valid syntax.
Do we have tests for these functions?

Reply:
It is valid because I made the method templated on the WORKING_RES parameter, right?  So you would call it like:
buildIndexSetAligned<camp::resources::Cuda>(....)    for example.

We have never had tests for these methods and I don't think anyone is using them.  They are something Jeff and I put together before RAJA was even a thing.

Reply:
Out of curiosity, is libBIS (libBuildIndexSet I think?) still a thing? Is anybody using that? I thought that was really clever when Jeff described it to me

Reply:
@MrBurmark you are correct. I screwed this up in more than one way. Working on fixes. Thanks for pointing out the fact that I shouldn't write code and push changes while on a webex.... 😞

Reply:
@MrBurmark I fixed the syntax and verified it compiles and works properly with a simple one-off code. We should probably discuss whether we want to keep these methods. I don't want to write tests for things we will discard.